asllib/Interpreter.ml

```diff
@@ -199,12 +200,19 @@ module Make (B : Backend.S) = struct
       | Some (AlreadyEvaluated v) -> v
       | Some (NotYetEvaluated e) ->
           let v =
-            try eval_expr e
+            try
+              eval_expr e
             with Error.ASLException e ->
-              if _warn then
+                  if _dbg || _warn then
                 Format.eprintf "@[<2>Ignoring static evaluation error:@ %a@]@."
                   Error.pp_error e;
               V_Int 0
+               | e ->
+                  if _dbg then
+                    Printf.eprintf
+                      "Evaluating constant %s failed with %s!"
+                      name (Printexc.to_string e) ;
+                  raise e
```

Comment:
Is that because of the famous problem with `1[63:0]`?
Is it fixed with the fixed sized `of_int`?

This catch-all feels very weird to me

Reply:
I kept it to be re-activated in case some other exception pops in. My fix is a patch.